src/adv.py

```diff
@@ -40,6 +43,17 @@
 earlier adventurers. The only exit is to the south.""",
         "s_to": "narrow",
     },
+    "wide": {
+        "name": "Wide Passage",
+        "description": "A wider passage leads you west. You smell cookies",
+        "n_to": "trap",
+        "e_to": "foyer",
+    },
+    "trap": {
+        "name": "Trap Room",
+        "description": """IT'S A TRAP! A Witch that is baking cookies 
```

Comment:
I love this! 👍 Do you want the paralysis to have an in-game effect? Does the player lose/die?

Reply:
I left the directions out of the code on purpose, so you get stuck there :)